config/database.yml.example

```diff
@@ -23,3 +25,72 @@ production:
   database: db/production.sqlite3
   pool: 5
   timeout: 5000
+
+#   PostgreSQL Configuration:
+#   gem install pg
+#
+#   Ensure the PostgreSQL 'pg' gem is defined in your Gemfile
+#   gem 'pg'
+#   Update with your credentials below for username and password.
+
+development:
+  adapter: postgresql
+  database: db/osem_dev_database
+  username: username
+  password: password
+  pool: 5
+  timeout: 5000
+
+test:
```

Comment:
In my opinion, just development is fine, we don't need to copy the same stuff for test and production.